form.yml.erb

```diff
@@ -179,7 +198,32 @@ attributes:
           data-min-ppn-pitzer: 1,
           data-max-ppn-pitzer: 48,
           data-option-for-owens: true,
-          data-option-for-pitzer: true
+          data-option-for-pitzer: true,
+          data-option-for-kubernetes: false
+        ]
+      - [
+          "pitzer",   "pitzer",
+          data-option-for-owens: false,
+          data-option-for-pitzer: false,
+          data-option-for-kubernetes: true
+        ]
+      - [
+          "owens",   "owens",
+          data-option-for-owens: false,
+          data-option-for-pitzer: false,
+          data-option-for-kubernetes: true
+        ]
+      - [
+          "pitzer gpu",   "pitzer-gpu",
+          data-option-for-owens: false,
+          data-option-for-pitzer: false,
+          data-option-for-kubernetes: true
+        ]
```

Comment:
Similar to the above, isn't this the  "48 gpu core" option?

Reply:
No, the nodes actually have 40 cores for the GPU nodes in Kubernetes.  The core count should be irrelevant for Kubernetes since it doesn't affect scheduling like it does with SLURM.